tests/manual/testpopupwindows.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no"/>
+        <script type="text/javascript">
+            function newPopup(url) {
+                setTimeout(() => popupWindow = window.open(url), 2000)
```

Comment:
```suggestion
                setTimeout(() => popupWindow = window.open(url), 2000);
```
On QML we do not have semi-colon at the end of JavaScript lines. However, on web context I have tried to add them.